types/trashable/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for trashable 1.0
+// Project: https://github.com/hjylewis/trashable#readme
+// Definitions by: fmiranda <https://github.com/filipemir>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line:export-just-namespace
+export = Trashable;
```

Comment:
You should use the [module function template](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) here since this is how the [package](https://unpkg.com/browse/trashable@1.0.7/lib/index.js) is exporting the function.